package.json

```diff
@@ -16,7 +16,10 @@
     "format": "prettier --write './src/**/*.{js,css,md}'"
   },
   "main": "build/bundle.js",
-  "dependencies": {},
+  "dependencies": {
+    "google-map-react": "1.1.4",
+    "lodash": "^4.17.0"
```

Comment:
not peerdeps?

Reply:
no: https://stackoverflow.com/questions/26737819/why-use-peer-dependencies-in-npm-for-plugins